src/nanoemoji/glue_together.py

```diff
@@ -0,0 +1,37 @@
+from fontTools.ttLib.tables import otTables as ot
+from fontTools import ttLib
+import os
+
+_ORIGINAL="/tmp/Bungee7.ttf"
+_COLR="/tmp/bungee7/AnEmojiFamily.ttf"
+
+font = ttLib.TTFont(_ORIGINAL)
+colr = ttLib.TTFont(_COLR)
+_OUTPUT = '/tmp/bungee7/Bungee_SVG_COLR.ttf'
+
+# Copy all glyphs used by COLR over
+_glyphs_to_copy = set()
+
+def _collect_glyphs(paint):
+  if paint.Format == ot.PaintFormat.PaintGlyph:
```

Comment:
and PaintColrGlyph too (we don't use them right now, but we may in the future)

Reply:
you're right, here you only care about glyphs that are in glyf, i.e. PaintGlyph only. PaintColrGlyph references are dealt with internally by COLR table.